### ISSUE TITLE
Option: Disable always on top

### ISSUE THREAD

**Issue by larshb** [ettoolong/PopupWindow#23]
Whenever I popup a tab it is becomes always on top.
I am using Firefox in Gnome, which gives every window an "Always on Top" option which is somehow overridden in these popups.
I don't know if it is a feature or a bug, but I can't find an option to disable always on top.

**Comment by signaleleven** [comment]
Same here. I use the same extension in Chrome on MacOS without this issue, so I assume it's specific to firefox. I wonder if it's even possible to fix this

**Comment by Fuseteam** [comment]
this seems like a duplicate of #11

**Comment by mikdmst1** [comment]
i need this option too.

**Comment by GitMensch** [comment]
Yes, that totally is a duplicate, please close and lets discuss it in #11.
